scenarios/docker-compose.yml

```
@@ -18,18 +18,20 @@ services:
     expose:
       - 8805
   mock-smf:
-    build:
-      context: ./topo
-      dockerfile: Dockerfile.scapy
+    image: ${PFCPSIM_IMAGE}
     hostname: mock-smf
     container_name: mock-smf
     tty: true
     stdin_open: true
+    profiles:
+      - start-mock-smf
```

Can you add a comment briefly explaining why we need this profile?
(if possible, I would use `mock-smf` as profile)
```suggestion
    profiles:
      - mock-smf
```

---

I was actually thinking of removing this profile since when creating the mock-smf container will just start the gRPC server in listening mode